docs/blog/2021-12-12-log4j-zero-day-mitigation-guide.mdx

```diff
@@ -428,12 +427,12 @@ _Spoiler: Maintained by us._
 
 An Open Source development framework that enables you to easily add a "Secure by Default" architecture to your existing
 software with only a few lines of code. It's loosely based on the OWASP Application Security Verification Standard 
-(above) and is also designed to be implemented in [multiple levels](https://www.lunasec.io/docs/pages/how-it-works/security/introduction/).
+(above), and is also designed to be implemented at [multiple levels](https://www.lunasec.io/docs/pages/how-it-works/security/introduction/).
 
 #### [Acra Database Protection](https://github.com/cossacklabs/acra)
 
 An Open Source database encryption library that helps you prevent data leaks by adding a layer of encryption to your 
-data. It's designed and maintained by Cossack Lab's, which also offers it as a commercial offering.
+data. It's designed and maintained by Cossack Labs, which also tenders it as a commercial offering.
```

Comment:
tenders? I think this should continue to say "offers" or say "sells"

Reply:
"offers as an offering" sounds weird and is redundant.   Tender means offer/sell, and sounds better than sells imo, but if you feel strongly I'll change it.